pipeline/transformer/transformer_sierra/src/main/scala/weco/pipeline/transformer/sierra/transformers/SierraGenres.scala

```diff
@@ -62,18 +60,36 @@ object SierraGenres
       }
       .distinct
 
+  private def identifyPrimaryConcept(varField: VarField): IdState.Unminted = {
+    // For Genres, the identifier found in $0 is assigned to the primary concept.
+    // This is in contrast to Subjects, where it is assigned to the Subject itself.
+    // However, for a subject, if no identifier is found, one is created that represents
+    // the whole field.  In Genres, this should not be the case, because the primary concept
+    // will get an identifier made for the concept itself on its own.
+    // This method fixes that inconsistency by discarding the LabelDerived identifier
+    // It's a bit hacky, but I hope it will go away at some point.
+    val wholeFieldConceptId = identifyConcept(
+      ontologyType = "Concept",
+      varField = varField
+    )
+    wholeFieldConceptId match {
+      case identifiable: IdState.Identifiable =>
+        if (identifiable.sourceIdentifier.identifierType == IdentifierType.LabelDerived)
+          IdState.Unidentifiable
+        else identifiable
+      case other => other
+    }
```

Comment:
Minor: you can put conditions on pattern matches, which is a bit more idiomatic:

```suggestion
    wholeFieldConceptId match {
      case identifiable: IdState.Identifiable if (identifiable.sourceIdentifier.identifierType == IdentifierType.LabelDerived) =>
        IdState.Unidentifiable

      case other => other
    }
```